app/main.py

```diff
@@ -1,3 +1,13 @@
 def cache(func):
-    # Write your code here
-    pass
+    old_results = {}
+
+    def inner(*args):
+        if args in old_results:
+            print("Getting from cache")
+            return old_results[args]
+        result = func(*args)
+        old_results[args] = result
+        print("Calculating new result")
+        return result
```

Comment:
see: 
```python
old_results[args] = result
```
that means that if you make `return result` it will be equal to `return old_results[args]`. But in the first case, you have already used `return old_results[args]`
So you make 2 the same returns. Make only 1 return for 2 cases